client/client.go

```diff
@@ -12,83 +15,98 @@ import (
 	"github.com/sajari/docconv"
 )
 
-var (
-	defaultParam    = "input"
```

Comment:
hmmm might be confusing with an old version. We def set this on v9. Anyway all good